app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,15 @@
+/*
+* This is a manifest file that'll be compiled into application.css, which will include all the files
```

Comment:
Is this not working?
```suggestion
  @import '/partials/variable';
```

Reply:
No, its giving me an error that it doesn't recognise the colour classes. 
![import_error](https://github.com/mla-wtag/ml_Journey/assets/141820916/5388b245-737a-4a42-abe9-6603a6045d87)

Reply:
Alright. Let's keep the current changes that you have for the variables. Let's investigate it on a follow-up story how we can centrally import them instead of doing so in all files.